pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -408,6 +412,10 @@ public void setRealtimeSegmentCommitTimeoutSeconds(int timeoutSec) {
     setProperty(SEGMENT_COMMIT_TIMEOUT_SECONDS, Integer.toString(timeoutSec));
   }
 
+  public void setControllerNumThreadPool(int numThreadPool) {
```

Comment:
Make the method name align with config, e.g. `setControllerExecutorNumThreads()`, same for other places